exo-go/src/application/runner.go

```diff
@@ -137,9 +141,11 @@ func (r *Runner) waitForOnlineText(cmdPlus *execplus.CmdPlus, role string, onlin
 	if err != nil {
 		fmt.Printf("'%s' failed to come online after an hour", role)
 	}
-	err = r.Logger.Log(role, fmt.Sprintf("'%s' is running", role), true)
-	if err != nil {
-		fmt.Printf("Error logging '%s' as online: %v\n", role, err)
+	if role != "" {
```

Comment:
Okay. Thoughts on updating this to have 
```
if role == "" { 
  return
}
```
That prevents the logging code from being indented an extra level